README.md

```diff
@@ -179,15 +179,15 @@ Besides specifying `DEFAULT_REPLY` in `slackbot_settings.py`, you can also decor
 
 ```python
 @default_reply
-def my_default_hanlder(message):
+def my_default_handler(messsage):
```

Comment:
This looks like a typo

Reply:
Ugh, I don't know where that typo came from but it's been haunting me on multiple commits/rebasing etc, guess I missed it again. I'll take care of that and change references in the README as well. I actually thought about that but didn't want to go gung-ho.